src/components/PasswordBox.js

```diff
@@ -36,11 +54,19 @@ class PasswordBox extends Nimiq.Observable {
         this._isInputValid = false;
 
         this.$el.addEventListener('submit', event => this._onSubmit(event));
+
+        if (options.showResetPassword) {
+            /** @type {HTMLLinkElement} */
```

Comment:
This is not a `<link>`.
```suggestion
            /** @type {HTMLAnchorElement} */
```